analysis.qmd

```diff
@@ -6,48 +6,59 @@ The draft currently has no official status. It is a work in progress and is subj
 
-## The Assurer's responsibilities during the analysis phase
+### The Assurer's responsibilities during the analysis phase
+
+The assurer shall review the assurance completed by the analyst, carry out any further validation and verification they may see as appropriate, and report errors and areas for improvement to the analyst. The assurer may then need to re-review the analytical work completed, as required. 
+
+
+### The Commissioner's responsibilities during the analysis phase
+
+The Commissioner’s role is to review any changes in design or methodology that the Analyst brings to their attention. The Commissioner should be available to provide input and clarifications to the Analyst. The aim is for the Commissioner to be aware of and accept changes to the design or methodology providing they meet their requirements. 
+
+### The Approver's responsibilities during the analysis phase
+
+The Approver should be aware of the progress of the analysis and ensure that they are available for approving the work at the Delivery phase.
 
-The analytical assurer shall review the assurance completed by the analyst, carry out any further validation and verification they may see as appropriate, and report errors and areas for improvement to the analyst. The assurer may then need to re-review the assurance completed, as required. 
+## Assurance during the analysis phase
 
-The assurer shall also confirm the work has been appropriately scoped, executed, validated, and verified and documented to the approver. 
+### Verification and validation
 
-## The Commissioner's responsibilities during the analysis phase
+TBC
 
-The commissioner’s role is to review the analysis and assurance as fit for the purposes of the commission.
+### Handling Uncertainty during the analysis phase
 
-## The Approver's responsibilities during the analysis phase
+While the Scoping and Design phases identified and described risks and uncertainties, the Analysis phase aims to assess and quantify the impact of uncertainty on the analytical outcome and their contribution to the range and likelihoods of possible outcomes. Guidance on how to identify, mitigate, measure and communicate uncertainty can be found in the [Uncertainty Toolkit for Analysts](https://analystsuncertaintytoolkit.github.io/UncertaintyWeb/index.html). The verification and validation by the analyst and assurer should assure the appropriate treatment of uncertainty.
 
-The Approver scrutinises the work of the analyst and assurer and shall be confident that the analysis:
+## Multi-use models and the analysis phase
 
-* Meets its design requirements;
-* Is of sufficient quality and
-* Is adequately documented.
+In multi-use models, analysis and edits may be carried out on individual elements of the model at differing times. This calls for mechanisms for assuring that the changes integrate into the larger model as expected, for example, through the use of test-suites.
```

Comment:
On multi use. I agree about the point on change control. 

Think that the role of the commissioner and approver should be restated and say that this function is normally done by a committee.